library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -464,17 +517,35 @@ public void setWeekDayTextAppearance(int resourceId) {
     }
 
     /**
-     * @return the currently selected day, or null if no selection
+     * @return the selected day, or null if no selection. If in multiple selection mode, this
+     * will return the last selected date
      */
     public CalendarDay getSelectedDate() {
```

Comment:
`getSelectedDate` should be refactored as `getLastSelectedDate` which makes more sense in a single vs multiple selection context